### PR TITLE
Bump Apache Pulsar version to 2.9.3

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

Bump Apache Pulsar version to 2.9.3

### Modifications

- Bump Apache Pulsar version to 2.9.3

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### CODE REVIEW

**Review by gaoran10 — APPROVED** [review]

File: charts/pulsar/Chart.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #
 
 apiVersion: v2
-appVersion: "2.9.2"
+appVersion: "2.9.3"
 description: Apache Pulsar Helm chart for Kubernetes
 name: pulsar
 version: 2.9.3
```
[the remark this answers is not in the document]
Comment (lhotari) [reply charts/pulsar/Chart.yaml:24]: There's PR #200 which improves the situation, but that stalled since I didn't receive feedback from @sijie .

**Review by lhotari — CHANGES_REQUESTED** [review]
you missed updating the tag `pulsar_metadata.image.tag`

**Comment by lhotari** [comment]
I guess tags referencing `2.9.2` should also be updated in `.ci/clusters/values-pulsar-image.yaml` file.

**Review by lhotari — CHANGES_REQUESTED** [review]
please remove the images and tags from `.ci/clusters/values-pulsar-image.yaml` so that it isn't necessary to update the file at all.

**Pull request merged**